### PR TITLE
pipes and redirects

### PR DESCRIPTION
**Pull request by MartimPinto**
Call me Mario, because I'm deep in these pipes

### CODE REVIEW

File: src/errors.c [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+/*   By: mcarneir <mcarneir@student.42.fr>          +#+  +:+       +#+        */
+/*                                                +#+#+#+#+#+   +#+           */
+/*   Created: 2023/11/23 13:05:23 by mcarneir          #+#    #+#             */
+/*   Updated: 2023/11/23 13:12:09 by mcarneir         ###   ########.fr       */
+/*                                                                            */
+/* ************************************************************************** */
+
+#include "../includes/minishell.h"
+
+extern int	g_status;
```
Comment (gabrrodr) [inline src/errors.c:15]: g_code

File: src/errors.c [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+int ms_error(int error)
+{
+	g_status = error;
```
Comment (gabrrodr) [inline src/errors.c:19]: code

File: src/execute.c [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+/*   By: mcarneir <mcarneir@student.42.fr>          +#+  +:+       +#+        */
+/*                                                +#+#+#+#+#+   +#+           */
+/*   Created: 2023/11/23 13:44:25 by mcarneir          #+#    #+#             */
+/*   Updated: 2023/11/23 16:31:12 by mcarneir         ###   ########.fr       */
+/*                                                                            */
+/* ************************************************************************** */
+
+#include "../includes/minishell.h"
+
+extern int	g_status;
```
Comment (gabrrodr) [inline src/execute.c:15]: code

**Review by gabrrodr — APPROVED** [review]
im glad my name isnt pipe then

**Review by MartimPinto — COMMENTED** [review]
funny

**Pull request merged**